docs/root/configuration/http/http_filters/bandwidth_limit_filter.rst

```diff
@@ -0,0 +1,64 @@
+.. _config_http_filters_bandwidth_limit:
+
+Bandwidth limit
+====================
+
+* Bandwidth limiting :ref:`architecture overview <arch_overview_bandwidth_limit>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`
+* This filter should be configured with the name ``envoy.filters.http.bandwidth_limit``.
+
+The HTTP Bandwidth limit filter limits the size of data flow to the max bandwidth set in the ``limit_kbps``
+when the request's route or virtual host has a per filter
+:ref:`bandwidth limit configuration <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`.
+
+If the bandwidth limit has been exhausted the filter stops further transfer until more bandwidth gets allocated
+according to the ``fill_interval`` (default is 50 milliseconds). If the connection buffer fills up with accumulated
+data then the source of data will have ``readDisable(true)`` set as described in the :repo:`flow control doc<source/docs/flow_control.md>`.
+
+.. note::
+  The token bucket is shared across all workers, thus the limits are applied per Envoy process.
+
+Example configuration
+---------------------
+
+Example filter configuration for a globally disabled bandwidth limiter but enabled for a specific route:
+
+.. literalinclude:: _include/bandwidth-limit-filter.yaml
+    :language: yaml
+    :lines: 11-53
+    :emphasize-lines: 9-29
+    :caption: :download:`bandwidth-limit-filter.yaml <_include/bandwidth-limit-filter.yaml>`
+
+Note that if this filter is configured as globally disabled and there are no virtual host or route level
+token buckets, no bandwidth limiting will be applied.
+
+Statistics
+----------
+
+The HTTP bandwidth limit filter outputs statistics in the *<stat_prefix>.http_bandwidth_limit.* namespace.
```

Comment:
this is kinda a literal i think - either way please make it:

```
 ``<stat_prefix>.http_bandwidth_limit.``
```

Reply:
sure